client/my-sites/plan-features-2023-grid/components/actions.tsx

```diff
@@ -201,6 +202,7 @@ const LoggedInPlansFeatureActionButton = ( {
 	selectedSiteSlug: string | null;
 	planActionOverrides?: PlanActionOverrides;
 } ) => {
+	const [ activeTooltipId, setActiveTooltipId ] = useState( '' );
```

Comment:
This state hook is duplicated both in here and in the plans comparison grid. Could we extract it into a self-contained hook so the code is [DRY-er](https://en.wikipedia.org/wiki/Don't_repeat_yourself)?